packages/sdks/overrides/react-native/src/functions/transform-block.ts

```diff
@@ -3,7 +3,7 @@ import type { BuilderBlock } from '../types/builder-block';
 export function transformBlock(block: BuilderBlock): BuilderBlock {
   // Map the DOM-based pixel format to a native compatible one.
   // react-native doesn't support DOM specific fields like `tagName` and `properties`, but any browser framework (e.g. Vue, etc.) do
-  if (block.id.startsWith('builder-pixel-')) {
+  if (block.id.startsWith('builder-pixel-') && !block.component) {
```

Comment:
Could you expand the comment above the code to explain why this check is needed? It isn't clear to me at face-value